x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/rule_event_log_list_table.tsx

```diff
@@ -290,6 +319,10 @@ export const RuleEventLogListTable = <T extends RuleEventLogListOptions>(
     [search, setSearchText]
   );
 
+  const onShowAllSpacesChange = useCallback(() => {
+    setShowFromAllSpaces(!showFromAllSpaces);
```

Comment:
nitpick: I think we can just simplify this `setShowFromAllSpaces((prev) => !prev))` etc